lib/bgs/auth_headers.rb

```diff
@@ -14,8 +14,8 @@ def to_h
 
     def auth_json
       {
-        external_uid: @user.uuid,
-        external_key: @user.email
+        external_uid: @user.ssn,
+        external_key: @user.ssn
```

Comment:
These `external_uid` and `external_key` values eventually get used [here](https://github.com/department-of-veterans-affairs/vets-api/pull/6113/files#diff-53edf2d59af06de8bd09bb8ac579f023e6c344fea20d08959cdd9fa55e3ce1f1L44) and I guess they really only exist in order to dump some unique identifiers into BGS' logging.

If we can simply just supply the `ssn` as those logging values, then I wonder if we even need this logic and this class anymore?  🤷   Perhaps we wouldn't even need [this line](https://github.com/department-of-veterans-affairs/vets-api/pull/6113/files#diff-975d964cd5f478b1004b6c518030143618a6cc108760b936d41ca37d1d54b3a5R39) anymore as well?  🤷 🤷‍♂️ 🤷‍♀️ 

I could easily be misunderstanding something though.

Reply:
we need those values in ClaimEstablisher
https://github.com/department-of-veterans-affairs/vets-api/blob/96bc7369a5187c618e0497d2a8f9835ec59f9321/modules/claims_api/app/workers/claims_api/claim_establisher.rb#L79

which, we should have the Veteran's ssn if this is guaranteed to be a representative submission, but because it could be the Veteran themself, those values need to get sent in there still i believe

Reply:
dude... i think youre right... i dont think this class is needed at all anymore :(

Reply:
Updated [here](https://github.com/department-of-veterans-affairs/vets-api/pull/6113/commits/a77c2cf001186ea1420f16eadaabc36c2dcddaea).